src/utils/index.js

```diff
@@ -0,0 +1,21 @@
+import childrenValueInputValidation from './childrenValueInputValidation';
+import createChainedFunction from './createChainedFunction';
+import createContextWrapper from './createContextWrapper';
+import CustomPropTypes from './CustomPropTypes';
+import deprecationWarning from './deprecationWarning';
+import domUtils from './domUtils';
+import EventListener from './EventListener';
+import assign from './Object.assign';
+import ValidComponentChildren from './ValidComponentChildren';
+
+export default {
+  childrenValueInputValidation,
+  createChainedFunction,
+  createContextWrapper,
+  CustomPropTypes,
+  deprecationWarning,
+  domUtils,
+  EventListener,
```

Comment:
That is the goal in the long run, but removing an internal usage is easier than removing something with explicit public exposure.